Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM golang
+MAINTAINER terranodo
+RUN mkdir -p /tegola/
+WORKDIR /tegola/
+COPY . /tegola
+
+#Requirements to compile 
+RUN mkdir -p /usr/local/go/src/github.com/terranodo/
+RUN cp -r /tegola/ /usr/local/go/src/github.com/terranodo/
+RUN mkdir -p /usr/local/go/src/github.com/BurntSushi/
```

Comment:
This step should not be necessary as we vendor the BurntSushi package. Are you having problems compiling without it? If so, I need to look into the vendor directory.